x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ingest_pipelines/ingest_pipelines_card.tsx

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
I think I lost some changes from upstream while resolving conflict. 

I will fix them and merge.